fe/fe-core/src/main/java/org/apache/doris/persist/DropResourceGroupOperationLog.java

```diff
@@ -0,0 +1,53 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.persist;
+
+import org.apache.doris.common.io.Text;
+import org.apache.doris.common.io.Writable;
+import org.apache.doris.persist.gson.GsonUtils;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+/**
+ * For resource group drop
+ */
+public class DropResourceGroupOperationLog implements Writable {
+    @SerializedName(value = "name")
+    private String name;
```

Comment:
I think it's better store id instead of name here, just like Drop a table(refer DropInfo.java).
 From the basic design logic of edit log, every LogEntry is immutable, and using append immuatble log way to implement mutable FE meta.
While tablename is a mutable property, this will introduce additional thinking costs.